### PR TITLE
Do not use hardcoded path for default HELMENV_ROOT

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

File: libexec/helmenv [hunk 1]
```diff
@@ -41,7 +41,7 @@ abs_dirname() {
 }
 
 if [ -z "${HELMENV_ROOT}" ]; then
-  HELMENV_ROOT="${HOME}/.helmenv"
+  HELMENV_ROOT="$(cd "$(dirname "$(readlink_f "${0}")")/.." && pwd)"
```
Comment (yuya-takeyama) [inline libexec/helmenv:58]: > readlink_f

What's this?
`readlink -f`?

**Review by yuya-takeyama — COMMENTED** [review]
Thanks for your pull request!

File: libexec/helmenv (hunk 1 above)
Comment (sagikazarmark) [reply libexec/helmenv:58]: To be honest, I copy pasted it from tfenv 😄

Comment (sagikazarmark) [reply libexec/helmenv:58]: Fixed!

**Comment by sagikazarmark** [comment]
Ping @yuya-takeyama

**Review by yuya-takeyama — APPROVED** [review]
Sorry to have kept you waiting.
I checked with both of Mac (macOS Mojave) and Linux (Ubuntu 18.04) and worked fine.

Thanks for your help!

**Pull request merged**